src/View/Helper/PaginatorHelper.php

```diff
@@ -820,9 +822,10 @@ protected function _firstNumber($ellipsis, $params, $start, $options)
     protected function _lastNumber($ellipsis, $params, $end, $options)
     {
         $out = '';
+        $last = is_int($options['last']) ? $options['last'] : 0;
```

Comment:
Yes, `1` & `'1'` will lead to 2 different kind of behavior.

It's needed if we want to use a custom template with a `'1'` - `'9'` as a value with the function `Paginator::numbers()` :
**Before the fix :**
```php
<?= $this->Paginator->numbers([
    'modulus' => 2,
    'first' => '1',
    'last' => (string)$this->Paginator->counter('{{pages}}'),
    'templates' => [
        'first' => '<li class="customFirst"><a href="{{url}}">{{text}}</a></li>',
        'last' => '<li class="customLast"><a href="{{url}}">{{text}}</a></li>'
    ]
]); ?>

// Will use the `first` template, but NOT the `last` template.
```
Result :
![534c5581afc60b2e5605fb4eb640bfff](https://cloud.githubusercontent.com/assets/8210023/11781881/8139873a-a26d-11e5-8616-4c2064409fbd.png)


**After the fix :**
```php
<?= $this->Paginator->numbers([
    'modulus' => 2,
    'first' => '1',
    'last' => (string)$this->Paginator->counter('{{pages}}'),
    'templates' => [
        'first' => '<li class="customFirst"><a href="{{url}}">{{text}}</a></li>',
        'last' => '<li class="customLast"><a href="{{url}}">{{text}}</a></li>'
    ]
]); ?>

// The `first` and `last` templates are used correctly.
```

Result :
![32ab540650126711e07a124e4d9b8cc4](https://cloud.githubusercontent.com/assets/8210023/11781850/4667c7e8-a26d-11e5-9799-62ffb9ec0100.png)